common/test/unit/com/thoughtworks/go/helper/P4TestRepo.java

```diff
@@ -110,7 +110,7 @@ public void stop() {
     }
 
     private ProcessWrapper startP4dInRepo(File tempRepo) throws IOException, CheckedCommandLineException {
-        CommandLine command = createCommandLine("p4d").withArgs("-C0", "-r", tempRepo.getAbsolutePath(), "-p", String.valueOf(port));
+        CommandLine command = createCommandLine("p4d").withArgs("-r", tempRepo.getAbsolutePath(), "-p", String.valueOf(port));
```

Comment:
The server couldn't start locally until I removed that option. Removing the option might be causing some of the perforce failures on build.gocd